### PR TITLE
Stats page update

### PR DESCRIPTION
**Pull request by OleanjiKingCode**
# Updating Stats Page in IQ Dashboard

_Rearranging the each stat in the page_

## How should this be tested?

1 Before 
image

2. Now
image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1302

### CODE REVIEW

File: src/components/client/StatsPage.tsx [hunk 1]
```diff
@@ -50,6 +50,11 @@ const StatsPage = () => {
     { label: 'EOS', value: data[prop]?.eos, icon: EOSLogo1 },
     { label: 'Polygon', value: data[prop]?.matic, icon: Polygon },
     { label: 'BSC', value: data[prop]?.bsc, icon: Bsc },
+    {
```
Comment (kesar) [inline src/components/client/StatsPage.tsx:53]: first item, not last

File: src/utils/use-stats-data.ts [hunk 2]
```diff
@@ -45,9 +45,9 @@ export function useStatsData() {
         volume: getMappedValue(volume.volume),
       }))
 
-      const hiiq = await getHiIQ()
+      const Iq = await getIQ()
       setData((prevState) => {
-        return { ...prevState, ...hiiq }
+        return { ...prevState, ...Iq }
```
Comment (Royal-lobster) [inline src/utils/use-stats-data.ts:50]: hmm, the iq can be all lowercased to follow variable convention

**Review by Royal-lobster — APPROVED** [review]
looks good !

File: src/utils/stats-data.ts [hunk 3]
```diff
@@ -97,12 +98,18 @@ const getTokenHolders = async () => {
     'https://ethplorer.io/service/service.php?data=0x579cea1889991f68acc35ff5c3dd0621ff29b0c9',
   )
   const data2 = await response2.json()
+
+  const response3 = await fetch(
+    'https://ethplorer.io/service/service.php?data=0x1bf5457ecaa14ff63cc89efd560e251e814e16ba&page=pageSize%3D600%26pageTab%3Dholders%26tab%3Dtab-holders&showTx=all',
+  )
+  const data3 = await response3.json()
```
Comment (Royal-lobster) [inline src/utils/stats-data.ts:105]: I think we can improve on variable names here too

**Pull request merged**